src/aap_eda/api/views/activation.py

```diff
@@ -374,6 +374,11 @@ def _get_activation_dependent_objects(self, activation):
             activation_id=activation["id"]
         )
         activation["instances"] = activation_instances
+        activation["restarted_at"] = (
```

Comment:
Hi @Dostonbek1 I think this logic is wrong. you are setting the field from the first instance, but if there is only one instance, the activation has not been restarted so this field should not have value

Reply:
Also, the restart count is only increased with automatic restarts, so returning a value when restart count is 0 (the activation has been disabled/enabled manually but it has more than one instance) would be confusing.

Reply:
https://github.com/ansible/eda-server/pull/385